addons/wowl/doc/services/router.md

```diff
@@ -72,3 +72,20 @@ in the url changed.
 
 When created, the router listens to every (external) hash changes, and trigger a
 `ROUTE_CHANGE` event on the main bus,
+
+## Redirect URL
+
+The `redirect` method will redirect the browser to `url`. If `wait` is true, sleep 1s and wait for the server i.e. after a restart.
```

Comment:
I'm not sure to understand the second sentence. I would say "... wait for the server (e.g. after a restart)."